[header design]

X4MU-L opened the pull request:
I added a css file and also linked it to the index.html file with the link tag in the head
I also added an index.js with which i was able to make some changes during a state in the file
I also added the HTML markup for the header and styled accordingly with the css file.
I have also imported the needed font file for this project.
Kindly go through and allow merge and give a go-ahead to continue project or advice on changes where needed.
Thanks

X4MU-L committed:
[header design]
I added a css file and also linked it to the index.html file with the link tag in the head
I also added an index.js with which i was able to make some changes during a state in the file
I also added the HTML markup for the header and styled accordingly with the css file.
I have also imported the needed font file for this project.
Kindly go through and allow merge and give a go-ahead to continue project or advice on changes where needed.
Thanks

EngrEric commented on index.css:
```diff
@@ -0,0 +1,136 @@
+@import url('https://fonts.googleapis.com/css2?family=Commissioner:wght@400;500;700&display=swap');
+
+*,*::before,*::after{
```
What are trying to achieve here

X4MU-L replied:
In the first line I was trying to import the fonts needed for the project...

In the third line I was trying box-size the everything so it doesn't overflow...(heard its a good practice)

EngrEric commented on index.css:
```diff
@@ -0,0 +1,136 @@
+
+
+
+
```
The empty space is not required

X4MU-L replied:
will have it resolved

EngrEric commented on index.html:
```diff
@@ -3,22 +3,35 @@
+  <!-- The Header Starts Here -->
+  <header>
+    <nav class="nav-links">
+      <div><img src="./images/logo.svg" alt="Crowdfunding Logo"></div>
```
Why the `div`?

X4MU-L replied:
more like a container....

EngrEric commented on index.js:
```diff
@@ -0,0 +1,15 @@
+const selectElements = selector =>{
+    const element = document.querySelector(selector);
+    if (element) return element;
+    else throw new Error(`something went wrong check if ${selector} is spelt correctly`)
```
You meant `not spelt correctly ...`

X4MU-L replied:
I actually meant if it is `spelt correctly` so of course `if not...` should be crosschecked and see what may be giving the `Error`

EngrEric reviewed (changes requested):
Minor changes required

X4MU-L committed:
[saved changes]

X4MU-L committed:
[added the CTA sections]
i have added the hero section of this design.
and also updated the necessary form sections

EngrEric reviewed (approved):
lgtm

X4MU-L committed:
[added BACKING modal as well as SUCCESS modal]
I added the HTML mark-up for the success modal and also made some changes to previous markup
I added added some JS functions to enable the user carry out some task on the page
I also made the page responsive on higher screens
In all also wrote some hopeful corresponding CSS for the challenge ���������
